velox/functions/prestosql/tests/JsonFunctionsTest.cpp

```diff
@@ -549,7 +549,7 @@ TEST_F(JsonFunctionsTest, jsonExtract) {
   EXPECT_EQ(
       "3", jsonExtract("{\"x\": {\"a\" : 1, \"b\" : [2, 3]} }", "$.x.b[1]"));
   EXPECT_EQ("2", jsonExtract("[1,2,3]", "$[1]"));
-  EXPECT_EQ(std::nullopt, jsonExtract("[1,null,3]", "$[1]"));
+  EXPECT_EQ("null", jsonExtract("[1,null,3]", "$[1]"));
```

Comment:
Here is what I see in Presto,

```
presto:di> select json_extract('[1,null,3]', '#[1]');
 _col0
-------
 NULL
(1 row)
```

Do you see something different?

Reply:
I run test case of `testJsonExtract()` in TestJsonExtractFunctions.java of Presto.

```
// pass
assertFunction(format("JSON_EXTRACT('%s', '%s')", "[1,null,3]", "$[1]"), JSON, "null");

// fail
assertFunction(format("JSON_EXTRACT('%s', '%s')", "[1,null,3]", "$[1]"), JSON, null);
```

Reply:
I see. Looks like I had a typo above:

```
presto:di> select json_extract('[1,null,3]', '$[1]');
 _col0
-------
 null
(1 row)
```